src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1618,6 +1624,8 @@ type Money {
 
 """"
 An ISO-8601-formatted duration, i.e. `PT2H30M` for 2 hours and 30 minutes.
+
+Negative durations are formatted like `PT-10M`.
```

Comment:
```suggestion
Negative durations are formatted like `-PT1H30M`.
```

Reply:
I do not recoment putting the `-` sign inside the duration, it may lead to confusing errors. `-PT1m3s` is 1 minute and 3 seconds, while `PT-1m3s` is 57 seconds.